app/Livewire/Project/Shared/ScheduledTask/Executions.php

```diff
@@ -26,4 +26,12 @@ public function selectTask($key): void
         }
         $this->selectedKey = $key;
     }
-}
+
+    public function getServerTimezone()
+    {
+        $server = data_get($this, 'destination.server');
```

Comment:
Here we also should get the destination server for the scheduled task but this is not available?